src/packages/@ncigdc/modern_components/ProjectsTable/ProjectsTable.js

```diff
@@ -78,16 +78,15 @@ export default compose(
                   </Tr>
                 ))}
                 <Tr>
-                  {tableInfo
-                    .filter(x => x.td)
-                    .map(
-                      x =>
-                        x.total ? (
-                          <x.total key={x.id} hits={hits} />
-                        ) : (
-                          <Td key={x.id} />
-                        ),
-                    )}
+                  {tableInfo.filter(x => x.td).map(x => {
+                    if (x.total) {
+                      return <x.total key={x.id} hits={hits} />;
+                    }
+                    if (x.id === 'project_id') {
+                      return <Td key="project_id">Total</Td>;
```

Comment:
is there a ticket to add this? I don't think this will make sense if you start moving the columns around

Reply:
is it possible to detect the id of the first displayed column and use that in the check/key

Reply:
but what if the first column needs to display an actual total like putting number of cases first. I think we should do something less hacky, not sure what though